### PR TITLE
Bindu, Supriya, Tarun | BAH-1222 | Address Translation gaps in the Clinical, IPD, OT and Document Upload modules

### PR DESCRIPTION
**Pull request by binduak**
Product JIRA card -> https://bahmni.atlassian.net/browse/BAH-1222

### CODE REVIEW

File: ui/app/bedmanagement/controllers/adtController.js [hunk 1]
```diff
@@ -187,7 +187,7 @@ angular.module('bahmni.ipd')
                 spinner.forPromise(bedService.assignBed(bed.bedId, patientUuid, encounterUuid).then(function () {
                     bed.status = "OCCUPIED";
                     $scope.$emit("event:patientAssignedToBed", $rootScope.selectedBedInfo.bed);
-                    messagingService.showMessage('info', "Bed " + bed.bedNumber + " is assigned successfully");
+                    messagingService.showMessage("info", $translate.instant("BED") + " " + bed.bedNumber + " " + $translate.instant("IS_SUCCESSFULLY_ASSIGNED_MESSAGE"));
```
Comment (buvaneswari-arun) [inline ui/app/bedmanagement/controllers/adtController.js:190]: Hardcoding "OCCUPIED"  can be moved to a config file.

Comment (binduak) [reply ui/app/bedmanagement/controllers/adtController.js:190]: yes @buvaneswari-arun I will make the change and push it to the same PR

File: ui/app/i18n/clinical/locale_en.json [hunk 2]
```diff
@@ -344,4 +344,20 @@
     "PROGRAM_PATIENT_STAGE": "Patient Stage",
     "CLINICAL_CLASS": "Class",
     "CLINICAL_CASTE": "Caste"
+    "NO_FORM_FOUND_FOR_PATIENT": "No Form found for this patient",
```
Comment (buvaneswari-arun) [inline ui/app/i18n/clinical/locale_en.json:347]: missing ' in this file

File: ui/app/bedmanagement/controllers/adtController.js (hunk 1 above)
Comment (binduak) [reply ui/app/bedmanagement/controllers/adtController.js:190]: @buvaneswari-arun after i gone through the code, i see that we are not using the bed.status field in the UI. Its just only for the verification purpose. 
But we can remove the hardcoded value with a constant but that would require changes in other place. I am thinking we can handle as part of the other card.

File: ui/app/i18n/clinical/locale_en.json (hunk 2 above)
Comment (binduak) [reply ui/app/i18n/clinical/locale_en.json:347]: Done @buvaneswari-arun

**Review by buvaneswari-arun — APPROVED** [review]

**Review by angshu — APPROVED** [review]

**Pull request merged**